lib/treat-locks/yarn.lock.js

```diff
@@ -47,9 +47,10 @@ async function run(folderPath, outputFolder, npmOptions = new NpmOptions()) {
       return false;
     }
     let content = '';
-    await fileUtil.readLineSync(filePath, async (line) => {
-      content = content.concat(treatLine(line, npmOptions));
-    });
+    const data = fs.readFileSync(filePath, 'utf8').split('\n');
```

Comment:
to remove line 49 and to replace the whole for by
```
const content = data.reduce((acc, curr) => `${acc}${treatLine(curr, npmOptions)}`, "")
```

Reply:
it could even be the case where L49 to L54 are totally replaced by just
```
fs.writeFileSync(`${output}`
                          , fs.readFileSync(filePath, 'utf8').split('\n')
                               .reduce((acc, curr) => `${acc}${treatLine(curr, npmOptions)}`, "")
);
```

but I guess it is less clear than just replacing the L49 + the `for`

Reply:
Yep I agree, L49 + the for loop change is clearer. I will update it